README.md

```diff
@@ -1 +1,11 @@
-Initial Commit
+This was made with create-react-app
+
+Steps to run:
+
+navigate from root to /my-app
```

Comment:
I submitted the challenge in this way because I just wanted you guys to see the overall file structure and that I used create-react-app. However, I agree that it would have been better to just submit`my-app`.